openrtb-json/src/main/scala/com/powerspace/openrtb/json/util/EncodingUtils.scala

```diff
@@ -42,8 +50,8 @@ object EncodingUtils {
       encoder.mapJson({
         json =>
           json.mapObject(_.mapValues {
-            case json: Json if json.isBoolean => Json.fromInt(json.asBoolean.map(_.toInt).get)
-            case json: Json => json
+            case boolean if boolean.isBoolean => Json.fromInt(boolean.asBoolean.map(_.toInt).get)
```

Comment:
my not match by type? 
case boolean: JsonBooleanObject => ...

Reply:
Unfortunately `JBoolean` is private.